awx/settings/defaults.py

```diff
@@ -883,6 +886,7 @@
         'awx.main.signals': {'level': 'INFO'},  # very verbose debug-level logs
         'awx.api.permissions': {'level': 'INFO'},  # very verbose debug-level logs
         'awx.analytics': {'handlers': ['external_logger'], 'level': 'INFO', 'propagate': False},
+        'awx.analytics.activity_stream': {'handlers': ['activity_stream'], 'level': 'DEBUG', 'propagate': False},
```

Comment:
I don't like either the level or the propagate here. That might misbehave.

Why? We need to propagate so that these logs make it to `awx.analytics` to get processed by the very important `external_logger` handler. If they don't propagate, it may stop going to the rsyslog/remote log aggregator.

the level probably wouldn't matter, but it's bad form. You don't want any level filter, because again, it might not make it up to the parent logger. In practice that would never happen because I know no logs going here will be below DEBUG level, but I still would rather allow _all_ logs through instead of putting a level filter in.

Reply:
I've followed the examples from the other loggers regarding the propagate option and the level was actually planned to be INFO. 
Based on your statement, should I set the propagate to true or add the external_logger to the handlers?

Reply:
no, I would just delete "level" and "propagate" here (in this logger). That's the best way to keep this targeted. That way, the _only_ thing it does is send logs going to this logger to the activity stream handler.